source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -215,28 +215,24 @@ void AMPseudoMotorControl::removeChildControl(AMControl *control)
 void AMPseudoMotorControl::onMoveStarted(QObject *action)
 {
 	Q_UNUSED(action)
-	qDebug() << name() << "move action started.";
```

Comment:
There are a couple more debugs in this file (setMoveInProgress() and setIsMoving()). Are these still required? If so can they be an AMErrorReport::Information or AMErrorReport::Debug, then the #include qDebug can be got rid of too.